articles/extensions/authorization-extension/v2/implementation/setup.md

```diff
@@ -0,0 +1,151 @@
+
+If you have a large number of users, managing the access rights and permissions each one individually can become unwieldy. The **groups** functionality helps make this process easier. For example, you might have your groups reflect the various departments of your organization: accounting, information technology, engineering, support, and so on. You might also create nested groups, such as dividing the engineering group into two nested groups: internal tools and client-facing applications. Your organization hierarchy therefore looks like this:
+
+* Corporation
+  * Accounting
+  * Information Technology
+  * Engineering
+    * Internal Tools
+    * Client-Facing Applications
+  * Support
+
+You can add users to your groups manually or dynamically based on the Connection(s) they're using to access your app. For example, if someone logs in using the Active Directory Connection and their AD profile indicates that they're in the Marketing group, the Authorization Extension can also add them to the Marketing group you're managing with the extension.
+
+Finally, we have permissions and roles, while are groups of permissions. The purpose of the latter is to make it easier to assign several permissions simultaneously to either a user or a group.
+
+For example, let's say that you want to grant permissions to:
+
+* Approve requests for travel
+* Approve travel expenses
+
+Rather than assigning both permissions to groups/users, you can roll the two (along with many others) into a role called **Travel Administrator**. You can then assign Travel Administrator to individual users or to one or more groups.
+
+## Users
+
+The **Users** section lists all the current users of your applications. You can use this to search for and select a specific user to see their profile, view or edit their group affiliations, and view or edit their roles.
+
+![Users Section](/media/articles/extensions/authorization/users.png)
+
+## Groups
+
+To create and manage the groups with which you'll manage users' settings, click **Groups** in the Authorization Dashboard.
+
+Click **Create Group** to create a new group for your users. You'll be asked to provide a **name** for the group, as well as a **description** for that group.
+
+![Create a New Group](/media/articles/extensions/authorization/create-group-v2.png)
+
+You can manage your users and their group affiliations in one of two ways:
+
+* Opening the **group** and managing the group's users
+
+    ![Open a Group](/media/articles/extensions/authorization/group-membership-v2.png)
+
+* Opening the **user** and managing the user's group membership(s)
+
+    ![Open a User](/media/articles/extensions/authorization/user-membership-v2.png)
+
+The groups you'll create are dependent on the needs of your business process. For example, you might have a group for your users in finance, a group for your users in IT, and so on. Additionally, you may create nested groups that are similar to the following:
+
+* Example Company
```

Comment:
(same comment on hierarchies and diagrams)
A diagram depicting the hierarchy would work so much better here. Can you address this in a separate PR? (since getting the diagram from the design team will take some time)